toolkit/templates/page-heading.html

```diff
@@ -4,7 +4,13 @@
       {{ context }}
     </p>
   {%- endif -%}
-  <h1>
-    {{ heading }}
-  </h1>
+  {%- if heading_level == '2' -%}
```

Comment:
The "page heading" macro is intended to be just that - the heading for a whole page - which is why it's always a h1. This makes sense to me as a toolkit macro - "this is how our top level heading on every page is styled", and that could easily change to be more than just the h1 element it is currently.

What's the need for a macro for h2 (and possibly lower) headings? The approach until now - to add heading styles to the `dmspeak` class - seems OK to me? 

If there is a need for more heading macros then I'd be tempted to make it a new macro - "paragraph_heading" or something, rather than mixing it up with the page heading.

Reply:
sorry - this got committed by accident, it was something I was testing out for another pull request. Has been removed.